.scrutinizer.yml

```diff
@@ -48,3 +48,10 @@ checks:
         instanceof_class_exists: true
         foreach_traversable: true
         no_unnecessary_function_call_in_for_loop: true
+tools:
+    php_cs_fixer:
```

Comment:
One word: No.

This type of code change needs to be done by a programmer that takes responsibility for making sure the automated tool worked correctly. A testing tool has no business changing the code.